.github/workflows/todo-check.yml

```diff
@@ -0,0 +1,18 @@
+name: TODO Checker
+
+on:
+    workflow_call:
+
+jobs:
+  find-todos:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check for TODOs in services directory         
+        uses: damienjburks/simple-todo-checker@1.0.0
```

Comment:
I am not familiar with the tool but always wary when I see someones idividual username in code/config.

Reply:
ah yes - i've taken the liberty of taking a chance at creating a todo checker in the marketplace; it seemed to be a common problem that should be solved.

Reply:
Agree with the hesitation from @sshiells-scottlogic— but creating items in a personal namespace is a fairly common practice for small tools. As long as we maintain a policy to pin versions, we can just do a code review of the external repo before upgrading.

Reply:
Yeah if it is an tool that you developed then that is all good (I have missed most of the recent meetings so I might have missed some context). I was just wanting to make sure that you weren't pointing to something local while testing that would then break when merged.